### PR TITLE
Describe how to change UI language by editing Geany shortcut

### PR DESCRIPTION
**Pull request by nomadbyte**
Addresses geany/geany#1757, describes the workarounds mentioned in the issue's discussion.

### CODE REVIEW

**Comment by eht16** [comment]
LGTM.
Tried the instructions on Windows and it worked as expected.

@elextr would mind checking with your legendary purple editing pencil? :D

File: page_content/documentation/faq.md [hunk 1]
```diff
@@ -86,21 +86,49 @@ for remote files, so there is no need to disable this setting with Geany 0.16.
...
+For example, to force an English UI (`en`) right-click to select the
+Shortcut's Properties and in the `Target` field put:
+
+```
+  cmd.exe /c "set ^"LANG=en^" & start /D ^"C:\installed-path\Geany\bin\^" geany.exe"
+```
+
+Adjust the `C:\installed-path\Geany` according to your Geany installation.
+Take care to put the `^"` (carrot-quote) as shown. This is the way to escape
+the quotes-within-the-quotes. There's a space before `geany.exe`.
```
Comment (elextr) [inline page_content/documentation/faq.md:102]: Make it "There is..." easier for non-english expert speakers.

File: page_content/documentation/faq.md [hunk 2]
```diff
@@ -86,21 +86,49 @@ for remote files, so there is no need to disable this setting with Geany 0.16.
...
-and put it into the `bin` subdirectory in the folder where you have Geany installed, 
-next to Geany.exe. Then open the file, edit the line `set LANG=C` and replace C 
-with your the language code of your locale (e.g. 'nl for Dutch, 'pt_BR' for 
-'Portuguese Brazilian'). Save the file and execute it. It should start Geany 
-with the desired language assuming there is an exising translation. 
+On Windows: The easiest way is to change the Geany shortcut that was created
+during the installation. It is recommended to create a copy of the Geany
+shortcut for the desired UI language.
+
+For example, to force an English UI (`en`) right-click to select the
```
Comment (elextr) [inline page_content/documentation/faq.md:93]: There are no previous instructions for this to be an example of, better to just make this an instruction to "Right click ...." and then say replace `en` with your language, as for `installed-path` and `C` below.

File: page_content/documentation/faq.md [hunk 3]
```diff
@@ -86,21 +86,49 @@ for remote files, so there is no need to disable this setting with Geany 0.16.
...
 `LANG=C geany`
 
-and of course, change "C" to your language code (see above) or set 
+and of course, change "C" to your language code (see above) or set
 your locale specific environment variables accordingly.
 
+If your system supports launcher shortcuts (for example, `.desktop` files),
+then Geany's shortcut can be updated (created a new one) to add the `LANG`
+setting as shown above. In case of `.desktop` files, this usually goes into
+`Exec` line.
```
Comment (elextr) [inline page_content/documentation/faq.md:130]: ...the `Exec` line.

File: page_content/documentation/faq.md [hunk 4]
```diff
@@ -86,21 +86,49 @@ for remote files, so there is no need to disable this setting with Geany 0.16.
 
 ## How can I change the language of the user interface?
 
-On Windows: Download https://download.geany.org/contrib/geany_english.bat 
-and put it into the `bin` subdirectory in the folder where you have Geany installed, 
-next to Geany.exe. Then open the file, edit the line `set LANG=C` and replace C 
-with your the language code of your locale (e.g. 'nl for Dutch, 'pt_BR' for 
-'Portuguese Brazilian'). Save the file and execute it. It should start Geany 
-with the desired language assuming there is an exising translation. 
+On Windows: The easiest way is to change the Geany shortcut that was created
```
Comment (elextr) [inline page_content/documentation/faq.md:89]: I'm not sure why this couldn't just be a sentence "On Windows the easiest ..." but I understand its the way it was before and would require changing other systems to be the same.

File: page_content/documentation/faq.md (hunk 2 above)
Comment (nomadbyte) [reply page_content/documentation/faq.md:93]: The section's question is "How to change the UI language?". The example here relates to the choice of language, `en` English that is.

File: page_content/documentation/faq.md (hunk 4 above)
Comment (nomadbyte) [reply page_content/documentation/faq.md:89]: I kept the original style for consistency.

File: page_content/documentation/faq.md (hunk 2 above)
Comment (nomadbyte) [reply page_content/documentation/faq.md]: I pushed the changes in a new commit.

**Comment by nomadbyte** [comment]
Thanks for your comments, I addressed them in two batches. The updated version is in commit [fa8f646]

**Comment by nomadbyte** [comment]
@eht16 , @elextr  Do you guys need any more input from me on this to get the PR going?

**Comment by eht16** [comment]
Fine by me.
I would like to have @elextr or @codebrainz to re-read it, I'm not a native speaker.

**Comment by eht16** [comment]
Thanks!

**Pull request merged**